### DISCUSSION TITLE
Login with alias

### DISCUSSION THREAD

**Discussion by dennmuel** [docker-mailserver/docker-mailserver#2706] [General]
Hi, first of all, thank you very much for your work on this! I really appreciate to have such a relatively easy to use mailserver at hand.

I have a question concerning login and alias addresses. I would have expected that you could only log in to your mail account with the "main" address, but not the alias addresses. However, in fact (at least with DMS 11.1.0) it seems possible to login in with any alias. As you do so, you do not see any of the messages or folders present in the account, you can however send messages. I've successfully done that from both Thunderbird and Roundcube.

Is this intended behavior and my expectation is just wrong? Or did I possibly misconfigure the server and if so, could you please point me to the correct settings?

(I'm not sure, if this is specific to DMS or generally email related, so please excuse, if this is a newbie question.)

Thanks in advance and best regards!
Dennis

**Comment by polarathene** [comment]
This is probably due to the current default [`ENABLE_QUOTAS` ENV](https://docker-mailserver.github.io/docker-mailserver/edge/config/environment/#enable_quotas) being enabled. If you set this to `0` to disable it, you should not be able to login via an alias.

This feature is [barely documented by us](https://docker-mailserver.github.io/docker-mailserver/edge/config/user-management/accounts/), more info can be read at the [Dovecot docs for Quota Plugin](https://doc.dovecot.org/configuration_manual/quota_plugin/#quota) and the [default config we provide for it](https://github.com/docker-mailserver/docker-mailserver/blob/57aeb6db2a6041e2ed4f243392315ad2b39bddb6/target/dovecot/90-quota.conf). AFAIK, it should be safe to disable the quotas feature if you don't need to set any mailbox storage limits. We may eventually switch the ENV default to disabled in a future major version release.

The reason this ENV is causing that is for that feature to work properly we have to use a [workaround with Dovecot that creates _dummy accounts_ for aliases](https://github.com/docker-mailserver/docker-mailserver/blob/57aeb6db2a6041e2ed4f243392315ad2b39bddb6/target/scripts/helpers/accounts.sh#L100-L107). That workaround creates those _dummy accounts_ with the [password for the aliased user](https://github.com/docker-mailserver/docker-mailserver/blob/57aeb6db2a6041e2ed4f243392315ad2b39bddb6/target/scripts/helpers/accounts.sh#L159), hence you can login with the alias. I would not rely on that functionality, as the workaround may eventually no longer be needed.

Mail receiving and sending is handled via Postfix, while Dovecot manages storage and mail clients:
- Dovecot provides IMAP and POP protocols for your mail client like Thunderbird and Roundcube to access mail, but these dummy accounts should not have any storage as that's received by Postfix and routed to the real mail account managed by Dovecot.
- While sending from your mail client will be handled with Postfix ports (_25 for SMTP submission, or if requiring authentication 587 or 465_) and sent out via one of those same ports depending on your configuration.

**Reply by dennmuel** [reply]
Hi @polarathene,

thanks for your quick and useful explanation! I disabled the quota feature and it's now behaving as I expected.

Best regards!
